provider/manifest/config.go

```diff
@@ -1,7 +1,5 @@
 package manifest
 
-import "time"
-
-type config struct {
-	ManifestLingerDuration time.Duration `env:"AKASH_MANIFEST_LINGER_DURATION" envDefault:"5m"`
+type ServiceConfig struct {
+	HTTPServicesRequireAtLeastOneHost bool
```

Comment:
`IngressExposeHostRequired`?  Just seems a bit verbose.

Reply:
Yeah, its verbose. I was just trying to be clear with it.